PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -101,6 +101,9 @@ function Export-AADProvider {
     # Read the properties and relationships of an authentication method policy
     $AuthenticationMethodPolicy = ConvertTo-Json @($Tracker.TryCommand("Get-MgBetaPolicyAuthenticationMethodPolicy"))
 
+    # 6.1
+    $DomainSettings = ConvertTo-Json @($Tracker.TryCommand("Get-MgBetaDomain"))    
```

Comment:
Remove trailing whitespace
```suggestion
    $DomainSettings = ConvertTo-Json @($Tracker.TryCommand("Get-MgBetaDomain"))
```

Reply:
Resolved in [492c112](https://github.com/cisagov/ScubaGear/pull/708/commits/492c11257cb4c115fb615aeb1589a535838f9fa9)